astroquery/mast/collections.py

```diff
@@ -162,7 +162,11 @@ def query_region_async(self, coordinates, *, radius=0.2*u.deg, catalog="Hsc",
         for prop, value in kwargs.items():
             params[prop] = value
 
-        return self._current_connection.service_request_async(service, params, pagesize=pagesize, page=page)
+        # Parameters will be passed as JSON objects only when accessing the PANSTARRS API
+        use_json = True if catalog.lower() == 'panstarrs' else False
```

Comment:
slight simplification; up to you whether you think the other version is easier to read
```suggestion
        use_json = catalog.lower() == 'panstarrs'
```

Reply:
Looks good!

Reply:
note there's one other instance like this below